misc/alert_rules.json

```diff
@@ -699,12 +699,12 @@
     "severity": "critical"
   },
   {
-    "rule": "storage.storage_deleted = 0 AND storage.storage_descr = \"/\" AND storage.storage_perc >= 90 AND storage.storage_perc < 95",
+    "rule": "storage.storage_deleted = 0 && storage.storage_descr = / && storage.storage_perc >= 90 && storage.storage_perc < 95",
```

Comment:
Are you sure you can remove the `"` enclosing `/` ? I would keep `\"/\"` for safety, this is a valid syntax.